source/tutorials/multi-boot/multi-boot.rst

```diff
@@ -0,0 +1,181 @@
+.. _multi-boot:
+
+Multi-boot |CL-ATTR| and Windows\* 10 OS
+########################################
+
+This guide shows how to install |CL-ATTR| adjacent to an existing Windows\*
+10 :abbr:`OS (operating system)`. To add |CL| to an existing Windows installation, there are two possible methods:
+
+.. contents::
+   :local:
+   :depth: 1
+
+Description
+***********
+
+In this tutorial you'll install |CL| :ref:`bare-metal-install-desktop` as
+an additional partition.
+
+Method 1: Shrink the Windows partition to make space for |CL|
+*************************************************************
+
+#. Boot up the Windows 10 OS.
+
+#. Launch the :file:`Disk Management` utility found under
+   :guilabel:`Create and format hard disk partitions`.
+
+#. Right-click the primary Windows partition and select “Shrink Volume...”.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-01.png
+      :width: 75%
+      :alt: Disk Management > Shrink Volume
+
+      Figure 1: Disk Management > Shrink Volume.
+
+#. Shrink the size of the partition by at least the following amount:
+
+   * For :ref:`bare-metal-install-desktop`, allow at least 21GB.
+
+   * For :ref:`bare-metal-install-server`, allow at least 4GB.
+
+#. We shrink C from about 91000 to 70000, as shown in Figure 2.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-02.png
+      :width: 75%
+      :alt: Shrink C
+
+      Figure 2: Shrink C.
+
+#. Shutdown the Windows 10 OS.
+
+#. Follow the instructions to :ref:`bare-metal-install-desktop`.
+
+#. After booting from the |CL| image, select the icon to
+   launch the installer.
+
+#. Click :guilabel:`Select Installation Media`.
+
+#. Select :guilabel:`Safe Installation`.
+
+#. Go through the remaining steps to install |CL|.
+
+#. Shut down your system and remove the USB.
+
+#. Reboot.
+
+#. During the BIOS POST stage, press the proper F-key for your system to
+   bring up the “Boot Menu”. We press :kbd:`F2`.
```

Comment:
The "Boot Menu" should look like this:

![IMG_4154](https://user-images.githubusercontent.com/29260389/66062602-92f30a80-e4f6-11e9-9fda-09b97349e275.jpg)

Your current screenshot is actually the BIOS.  

The "Boot Menu" is a quick way to select a partition to boot.  The BIOS "Boot Order" is a way to set the preferred order.  

If you want to take the latter approach, the just change the wording from "Boot Menu" to "Boot Order".  

My original preference was to use the "Boot Menu" to show users they can select either Clear or Windows to boot instead of relying on the Clear bootloader.